internal/inventory/tagprocessor_test.go

```diff
@@ -22,6 +22,9 @@ func getTestingLogger() logger.LoggingClient {
 	logLevel := "WARN"
 	if testing.Verbose() {
 		logLevel = "DEBUG"
+		log := logger.NewMockClient()
+		log.SetLogLevel(logLevel)
+		return log
```

Comment:
This is opposite of what is needed. Should be `return logger.NewClientStdOut("test", false, logLevel)` here and `return logger.NewMockClient()`. i.e. 

```
       if testing.Verbose() {
		return logger.NewClientStdOut("test", false, "DEBUG")
	}

	return  logger.NewMockClient() 
``` 
Also logLevel := "WARN" on line 22 can be removed.
This way when verbose is enable we get the DEBUG log message, otherwise is is silent.